MapboxNavigation/NavigationViewController.swift

```diff
@@ -148,16 +148,29 @@ open class NavigationViewController: UIViewController, NavigationStatusPresenter
     public var shouldManageApplicationIdleTimer = true
     
     /**
-     Enables highlighting of the destination building when arriving.
+     Enum denoting the types of the destination building highlighting on arrival.
      */
-    public var highlightsDestinationBuildings: Bool = false
+    public enum DestinationBuildingHighlightType: Int {
+        /**
+         Do not highlight destination building on arrival.
```

Comment:
Perhaps say something here about the default fallback. I think we still fall back to displaying a 📍, right?

Reply:
Yep, added default fallback note.